### ISSUE TITLE
Begging for Apple silicon M1 custom

### ISSUE THREAD

**Issue by bigNeoneo** [atom/atom#23500]
my laptop is mac with M1 chip, an arm-based chip, although can run Atom with rosetta2, I still assume will it better faster or more effcient run as original arm environment?

**Comment by Immortal-Hak3r** [comment]
yes

**Comment by xyzzy529** [comment]
As I understand it, the Atom code is primarily JavaScript ... which is interpreted.
So the program should run at full speed once you install it based on the MacOS instructions.
Please let me know if there is another consideration for MacOS.

**Comment by asturur** [comment]
I think the issue is that the browser wrapper, electron is m1 optimized only from electron 11 onward.

**Comment by icecream17** [comment]
We're on electron 11 so it's possible

And here's a link to the main issue? https://github.com/atom/atom/issues/21688
https://github.com/atom/atom/issues?q=is%3Aissue+is%3Aopen+apple+silicon
Tracking support: https://github.com/atom/atom/discussions/22023

**Comment by asturur** [comment]
Yes, but the official release is still on 1.58, while electron 11 has been introduced with the current 1.61 cycle right? I'm not sure what will get out of beta and when. Is it actually possible to public arm64 from the ci? are there any github runners for that?

**Comment by lockieluke** [comment]
god of atom please 🙏🥺

**Comment by Ulisses1478** [comment]
Im using vscode for months now good news,  I'll dump as soon as I see the update

**Comment by bogdancss** [comment]
https://github.blog/2022-06-08-sunsetting-atom/

Atom is dead it seems

**Comment by asturur** [comment]
With MS buying github this was unevitable. 
I wish they had contributed to atom from the start instead of creating VS code.

**Comment by lockieluke** [comment]
[https://github.com/atom-community/atom](https://github.com/atom-community/atom)

there's a fork
